tests/HTMLPurifierTest.hack

```diff
@@ -137,6 +214,9 @@ class HTMLPurifierTest extends HackTest {
 	public function testDOM(): void {
 		echo "\nrunning testDOM()...";
 		$config = HTMLPurifier\HTMLPurifier_Config::createDefault();
+		$policy = HTMLPurifier\HTMLPurifier_Policy::fromDefault();
+		// This is weird, but based off of how we define config in HTMLPurifier constructor
+		$config = $policy |> $$->configPolicy($config);
```

Comment:
I think if we just did `$policy->configPolicy($config);` it may be more clear than the piping.

Reply:
I think that was a remnant from a find/replace all I did... I totally agree, and I'll double check to see if there's anything else like this.